.pipelines/cosmos-pipelines.yml

```diff
@@ -32,6 +32,12 @@ variables:
   buildPlatform: 'Any CPU'
   buildConfiguration: 'Release'
 steps:
+- task: CmdLine@2
+  displayName: 'Set flag to publish Verify *.received files when tests fail'
+  condition: failed()
+  inputs:
+    script: 'echo ##vso[task.setvariable variable=publishverify]Yes'
```

Comment:
wouldn't this need to be placed after the tests run so we have a valid failure condition to detect? e.g. if this is step 1, would this ever enter the failure condition?

Reply:
You'd assume that after having read the docs _multiple_ times I'd have noticed that it said it should go after the tests were run not at the top of the steps list but nooooooooooooooo.

Fixed across all pipelines.